Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -307,6 +314,12 @@ private bool IsTooFarFromWall()
             
             return true;
         }
+
+        private bool IsNextWallAngleTooObtuse()
+        {
+            var angleDifference = Vector3.SignedAngle(ChildTransform.forward, _lastPlayerWallRunDirection, Vector3.up);
+            return angleDifference == 0;
```

Comment:
since `angleDifference` returns a float, maybe we want to use `Mathf.Approximately(angleDifference, 0.0f)` to take into account floating pointers (https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html).